### ISSUE TITLE
Fix warnings against latest Jellyfish

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-jellyfish#2]
```
tests/test_jellyfish.py::test_jellyfish[jaro_distance("hello", "hello world")-expected8]
  /Users/simon/.local/share/virtualenvs/datasette-jellyfish-PkdpuL5R/lib/python3.7/site-packages/jellyfish/__init__.py:26: DeprecationWarning: the jaro_distance function incorrectly returns the jaro similarity, replace your usage with jaro_similarity before 1.0
    DeprecationWarning,

tests/test_jellyfish.py::test_jellyfish[jaro_winkler("hello", "hello world")-expected9]
  /Users/simon/.local/share/virtualenvs/datasette-jellyfish-PkdpuL5R/lib/python3.7/site-packages/jellyfish/__init__.py:17: DeprecationWarning: the name 'jaro_winkler' is deprecated and will be removed in jellyfish 1.0, for the same functionality please use jaro_winkler_similarity
    DeprecationWarning,
```

**Comment by simonw** [comment]
https://jellyfish.readthedocs.io/en/latest/changelog.html#may-2020

> 0.8.2 - 21 May 2020
> -------------------
> 
> -   fix jaro_winkler/jaro_winkler_similarity mix-up
> -   deprecate jaro_distance in favor of jaro_similarity backwards compatible shim left in place, will be removed in 1.0
> -   (note: 0.8.1 was a broken release without proper C libraries)
> 
> 0.8.0 - 21 May 2020
> -------------------
> 
> -   rename jaro_winkler to jaro_winkler_similarity to match other functions backwards compatible shim added, but will be removed in 1.0
> -   fix soundex bug with W/H cases, #83
> -   fix metaphone bug with WH prefix, #108
> -   fix C match rating codex bug with duplicate letters, #121
> -   fix metaphone bug with leading vowels and 'kn' pair, #123
> -   fix Python jaro_winkler bug #124
> -   fix Python 3.9 deprecation warning
> -   add manylinux wheels

**Comment by simonw** [comment]
Since I haven't hit 1.0 yet I'm going to change the names of the SQL functions to reflect these updates.

Will also pin to minimum version of 0.8.2
